tests/testthat/testS4sort/DESCRIPTION

```diff
@@ -0,0 +1,11 @@
+Package: testS4sort
+Title: Test package for sorting S4 classes
+License: GPL-3
```

Comment:
I think it would be best to just take out the `License` field here. Or it should be the same license as devtools itself.